src/Backups/BackupIO_File.cpp

```diff
@@ -139,9 +139,9 @@ DataSourceDescription BackupReaderFile::getDataSourceDescription() const
 }
 
 
-bool BackupWriterFile::supportNativeCopy(DataSourceDescription data_source_description) const
+bool BackupWriterFile::supportNativeCopy(DataSourceDescription data_source_description, bool has_throttling) const
 {
-    return data_source_description == getDataSourceDescription();
+    return !has_throttling && data_source_description == getDataSourceDescription();
```

Comment:
"Native" copy functions can sometimes call not native implementations, could you move this check to copy functions? Because here it doesn't look good for me.

Reply:
Ok, done.